### PR TITLE
[stable/traefik] Add Image Pull Policy

### PR DESCRIPTION
**Pull request by Ealenn**
#### What this PR does / why we need it:

This PR allow to setup Image Pull Policy with helm values. 

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by Ealenn** [comment]
✅ DCO Check

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Default values for Traefik
 image: traefik
 imageTag: 1.7.20
+imagePullPolicy: IfNotPresent
```
Comment (dtomcej) [inline stable/traefik/values.yaml:4]: Can you comment this out, so that it is not required by default?

**Comment by cpanato** [comment]
@Ealenn can you update your PR and fix the conflicts? thanks!

File: stable/traefik/values.yaml (hunk 1 above)
Comment (Ealenn) [reply stable/traefik/values.yaml:4]: I don't understand, you don't want to keep `IfNotPresent` by default ?

Comment (cpanato) [reply stable/traefik/values.yaml:4]: we can keep, it makes sense

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Pull request merged**